tests/installation/addon_products_via_SCC_yast2.pm

```diff
@@ -28,7 +28,7 @@ sub run {
         script_sudo "echo \"@addon_proxy.proxy.scc.suse.de\" > /etc/SUSEConnect", 0;
     }
     type_string "killall xterm\n";
-    x11_start_program("xdg-su -c '/sbin/yast2'", target_match => 'yast2-control-center-ui');
+    x11_start_program("xdg-su -c '/sbin/yast2'", valid => 0 );
```

Comment:
Please, consider launch_yast2_module_x11 function from y2x11test: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/y2x11test.pm#L16
It's a wrapper for yast modules, which includes password typing logic. But packagekit-warning logic is not in there, but it doesn't hurt to move it there.